content/02.intro.md

```diff
@@ -1 +1,10 @@
 ## Introduction
+
+Deep learning (DL), a subfield of machine learning (ML), is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
+However, DL is large and complex field, and its correct application to biological research remains a daunting task for most.
+While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+
+To fix this problem, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-review] and Manubot [@url:https://greenelab.github.io/meta-review/].
+In the course of our discussions, several themes became clear: the importance of understanding and applying ML fundamentals, the necessity for extensive model validation and tuning, and the need for critical thought in interpreting results generated by means of DL, among others. 
+The rules we established range from high-level guidance to implementation best practices, and it is our hope that these rules will  provide actionable, DL-specific advice for both new and experienced DL practitioners alike.
```

Comment:
"The rules we established range from high-level guidance to implementation best practices, and it is our hope that these rules will  provide actionable, DL-specific advice for both new and experienced DL practitioners alike." 

=>

The rules we established range from high-level guidance to the implementation of best practices, and it is our hope that these rules will  provide actionable, DL-specific advice for both new and experienced DL practitioners alike who would like to employ DL in biological research.